vmdb/db/migrate/20150324111111_add_retirement_requester_to_vms_and_services.rb

```diff
@@ -0,0 +1,7 @@
+class AddRetirementRequesterToVmsAndServices < ActiveRecord::Migration
+  def change
+    [:vms, :services].each do |klass|
```

Comment:
This loop is overkill.  It would be a lot clearer to just have two lines.  Additionally, if a particular line fails, finding which one is the culprit is harder with a loop.